modules/promotions/admin-menu-items/custom-code-promotion-item.php

```diff
@@ -19,6 +19,17 @@ public function get_promotion_title() {
 		return esc_html__( 'Add Your Custom Code', 'elementor' );
 	}
 
+	public function get_promotion_description() {
+		return esc_html__(
+			'Custom Code is a tool gives you one place where you can insert scripts, rather than dealing with dozens of different plugins and deal with code.',
+			'elementor'
+		);
+	}
+
+	/**
+	 * @deprecated use get_promotion_description instead
+	 * @return void
+	 */
 	public function render_promotion_description() {
 		echo esc_html__(
 			'Custom Code is a tool gives you one place where you can insert scripts, rather than dealing with dozens of different plugins and deal with code.',
```

Comment:
@Barcov 

```
public function render_promotion_description() {
     echo $this->get_promotion_description();
}
```

Otherwise you are adding the text twice.

Reply:
Honestly I think these changes should not have made it into this PR in the first place. This changes is exactly like you asked for earlier today. 
<img width="1070" alt="image" src="https://github.com/elementor/elementor/assets/130742176/b3575e62-7ad5-43ca-994e-b89b7c5af137">

So we can go around again but it's wasting valuable company time.

Reply:
@Barcov What do you mean? We could not just totally delete a public function 🤔